Modules/ITS/src/TrendingTaskITSError.cxx

```diff
@@ -0,0 +1,275 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+///
+/// \file    TrendingTaskITSError.cxx
+/// \author  Mariia Selina on the structure from Piotr Konopka
+///
+
+#include "ITS/TrendingTaskITSError.h"
+#include "ITS/ReductorBinContent.h"
+
+#include "QualityControl/RootClassFactory.h"
+#include "QualityControl/DatabaseInterface.h"
+#include "QualityControl/MonitorObject.h"
+#include "QualityControl/QcInfoLogger.h"
+#include "QualityControl/Reductor.h"
+#include "QualityControl/ObjectMetadataKeys.h"
+
+#include <TList.h>
+#include <TObject.h>
+#include <TLegendEntry.h>
+#include <TCanvas.h>
+#include <TH1.h>
+#include <TGraph.h>
+#include <TMultiGraph.h>
+#include <TFile.h>
+#include <TDatime.h>
+#include <map>
+#include <string>
+#include <TTreeReader.h>
+#include <TTreeReaderValue.h>
+#include <TTreeReaderArray.h>
+
+using namespace o2::quality_control;
+using namespace o2::quality_control::core;
+using namespace o2::quality_control::postprocessing;
+using namespace o2::quality_control::repository;
+using namespace o2::quality_control_modules::its;
+
+void TrendingTaskITSError::configure(std::string name,
+                                     const boost::property_tree::ptree& config)
+{
+  mConfig = TrendingTaskConfigITS(name, config);
+}
+
+void TrendingTaskITSError::initialize(Trigger, framework::ServiceRegistryRef)
+{
+  // Preparing data structure of TTree
+  mTrend = std::make_unique<TTree>();
+  mTrend->SetName(PostProcessingInterface::getName().c_str());
+  mTrend->Branch("runNumber", &mMetaData.runNumber);
+  mTrend->Branch("Time", &mTime);
+  mTrend->Branch("Entries", &nEntries);
+
+  for (const auto& source : mConfig.dataSources) {
+    std::unique_ptr<ReductorBinContent> reductor(root_class_factory::create<ReductorBinContent>(source.moduleName, source.reductorName));
+    reductor->setParams(22, 0);
```

Comment:
Could you define "22" as some "const int" variable in the header? 

This 22 corresponds to the number of error plots provided by your code